### DISCUSSION TITLE
"Resource not accessible by integration"

### DISCUSSION THREAD

**Discussion by xero-lib** [lowlighter/metrics#1564] [Q&A]
I've tried with a token that has every scope, yet I get this error with most configurations besides what's currently on my profile.

I tried directly copying lowlighter's workflow and received this error.

**Comment by seanpm2001** [comment]
I am having the exact same issue. Can we get some attention on this issue?

**Comment by night0721** [comment]
Go to Repository settings->Actions->General, scroll to bottom, at workflow permission, turn on "Read and write permissions" instead of "Read repository contents and packages permissions"
